operator/pkg/cli/kafka.go

```diff
@@ -86,11 +88,30 @@ func NewKafkaClient(kafkaBroker string, kafkaBrokerIsSet bool, schedulerHost str
 		"auto.offset.reset": "largest",
 	}
 
+	namespace := DefaultNamespace
+
 	if config.Kafka != nil {
-		consumerConfig["security.protocol"] = "ssl"
-		consumerConfig["ssl.ca.location"] = config.Kafka.CaPath
-		consumerConfig["ssl.key.location"] = config.Kafka.KeyPath
-		consumerConfig["ssl.certificate.location"] = config.Kafka.CrtPath
+		if config.Kafka.Namespace != "" {
+			namespace = config.Kafka.Namespace
+		}
+		if config.Kafka.Protocol == "ssl" {
```

Comment:
nit: should we have an enum for ` config.Kafka.Protocol`?

Reply:
I'll change to some constants for now. These would need to match the available values used in kafka.